lms/djangoapps/class_dashboard/tests/test_dashboard_data.py

```diff
@@ -85,6 +98,36 @@ def setUp(self):
                     state=json.dumps({'attempts': self.attempts}),
                 )
 
+            # adding an instructor and a staff to the site. They should not be counted. Test should not fail
+            StudentModuleFactory.create(
+                grade=1,
+                max_grade=1,
+                student=instructor_member,
+                course_id=self.course.id,
+                module_state_key=self.item.location,
+                state=json.dumps({'attempts': self.attempts}),
+            )
+
+            StudentModuleFactory.create(
+                grade=1,
+                max_grade=1,
+                student=staff_member,
+                course_id=self.course.id,
+                module_state_key=self.item.location,
+                state=json.dumps({'attempts': self.attempts}),
+            )
+
+            # Adding this student will fail the test
+
+            # StudentModuleFactory.create(
+            #     grade=1,
+            #     max_grade=1,
+            #     student=newstudent,
+            #     course_id=self.course.id,
+            #     module_state_key=self.item.location,
+            #     state=json.dumps({'attempts': self.attempts}),
+            # )
+
```

Comment:
I think one test you can add, just to sanity check, is that the number of `StudentModules` belonging to `staff_member` and also to `instructor_member`, is 1.  

B/C unfortunately we can only check in tests that are supposed to pass, the commented-out test is good for code review but not for the automated testing =)